pkg/plugin/resources.go

```diff
@@ -452,13 +448,25 @@ func (app *App) handleSaveLLMOptInState(w http.ResponseWriter, req *http.Request
 
 	// write a success response body since backendSrv.* needs a valid json response body
 	w.WriteHeader(http.StatusOK)
-	_, err = w.Write([]byte(`{"status": "Success"}`))
+	_, _ = w.Write([]byte(`{"status": "Success"}`))
 	if err != nil {
 		handleError(w, fmt.Errorf("failed to write response body %w", err), http.StatusInternalServerError)
```

Comment:
Did you want to remove this branch then too?

Reply:
Ahh yep. After writing a success header you can't re-write the header to a failure so not much we can do.